docs/faq.rst

```diff
@@ -0,0 +1,13 @@
+.. _faq:
+
+Frequently Asked Questions
+==========================
+
+What is the difference between a Job and a Worker?
+--------------------------------------------------
+
+In dask-distributed, a ``Worker`` is a Python object and node in a dask
+``Cluster`` that serves two purposes, 1) serve data, and 2) perform\
+computations. Jobs are resources submitted to, and managed by, the job queueing
+system (e.g. PBS, SGE, etc.). In dask-jobqueue, a single Job may include one or
+more Workers.
```

Comment:
We might put this information in the dostring of the most common function that users might see the term "job".

I'm somewhat against having a FAQ if it only has this one question in it.  This seems more like a section of "stuff I think you should know but couldn't find a place to put in the docs" rather than frequently asked questions.

Reply:
And we should maybe point this part of the doc to the example that @mrocklin rightfully pointed to nicely informative!

Reply:
I've removed the FAQ.